models/stg_quickbooks__account.sql

```diff
@@ -25,10 +25,10 @@ account as (
 final as (
 
     select
-        cast(id as {{ dbt_utils.type_int() }}) as account_id,
+        cast(id as {{ dbt_utils.type_string() }}) as account_id,
```

Comment:
For my learning, why do we have to explicitly cast here as well although we have specified in the macro that this column (and all other ID columns) should be of type string?

Reply:
The macro only applies the casting if the field doesn't exist. If the field does exist then it will just use whatever the inferred datatype from the source is. Because of that, we have to define here as well for the fields that do exist.